staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go

```diff
@@ -23,6 +23,12 @@ import (
 	testingclock "k8s.io/utils/clock/testing"
 )
 
+var NewFakeBackOff = NewBackOffWithClock
```

Comment:
why export this? it's unusable by things outside this package

Reply:
```sh
$ find . -type f | xargs bat
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: ./go.mod
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ module example.com
   2   │ 
   3   │ go 1.19
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: ./a/a_test.go
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ package a
   2   │ 
   3   │ import (
   4   │     "fmt"
   5   │     "testing"
   6   │ 
   7   │     "example.com/b"
   8   │ )
   9   │ 
  10   │ func TestA(t *testing.T) {
  11   │     fmt.Println(b.Exported)
  12   │ }
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: ./b/b_test.go
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ package b
   2   │ 
   3   │ var Exported = "Hello, world"
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: ./b/b.go
───────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ package b
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

$ go test ./a
# example.com/a [example.com/a.test]
a/a_test.go:11:16: undefined: b.Exported
FAIL	example.com/a [build failed]
FAIL
```

Reply:
I was trying to reduce the diff, nothing else, let me change it

Reply:
I removed it now